modules/ROOT/pages/viewing-api-details.adoc

```diff
@@ -13,28 +13,45 @@ Name of the API.
 * *Type*
 + 
 The API type can be AsyncAPI, HTML, REST, or SOAP.
-* *Description*
-+
-A description of the API.
 * *Contact email*
 +
 The email of the contact for the API.
-* *Version*
+
+* *Categories*
++
+The name and value of the predefined category this API belongs to. APIs can have multiple categories. Categories are used to organize APIs into groups to improve browsing and discovery. Categories are created and maintained in Exchange.
```

Comment:
Is there any way to make these parallel - i.e., all sentence fragments or complete sentences?

Reply:
Any way to use active voice here?
Use categories...?
Create categories...?